src/main/java/tests/RepresentativesPageValidation.java

```diff
@@ -1,4 +1,22 @@
 package tests;
 
+import domain.pages.RepresentativesPage;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+
 public class RepresentativesPageValidation {
+
+    @Test
+    public void verifyRepresentativesPageLogo(){
+        System.setProperty("WebDriver.chrome.driver", "resources/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        RepresentativesPage representativesPage = new PageFactory().initElements(driver, RepresentativesPage.class);
+
+        driver.get("https://www.house.gov");
+        driver.findElement(By.linkText("Representatives")).click();
```

Comment:
create this click method in domain (Homepage) and call it here using a homePage object